corehq/apps/app_manager/templates/app_manager/v1/partials/case_list_module_overwrite.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load xforms_extras %}
+{% load app_manager_extras %}
+{% if module.module_type != 'basic' or request|toggle_enabled:'COPY_CASE_CONFIGS' %}
+<form class="form-inline" method='POST' action='{% url "overwrite_module_case_list" domain app.id module.id %}'>
+    {% csrf_token %}
+    <fieldset>
```

Comment:
one of the reasons for a v2 template is that this styling is updated to a panel for the settings pages. so you're not just copying in vain either.